x/params/keeper.go

```diff
@@ -21,7 +21,7 @@ type Keeper struct {
 }
 
 // NewKeeper constructs a params keeper
-func NewKeeper(cdc *codec.Codec, key *sdk.KVStoreKey, tkey *sdk.TransientStoreKey, codespace sdk.CodespaceType) (k Keeper) {
+func NewKeeper(cdc *codec.Codec, key sdk.StoreKey, tkey sdk.StoreKey, codespace sdk.CodespaceType) (k Keeper) {
```

Comment:
I'm ok with the `KVStoreKey -> StoreKey` change, not sure about the `TransientStoreKey -> StoreKey` one

Reply:
why the change on transient key?

Reply:
``tkey` can be `github.com/cosmos/cosmos-sdk/store.StoreKey` (interfacer)` I can no lint it if its requested